src/stack.ts

```diff
@@ -2376,5 +2385,30 @@ export class Stack {
       paths,
     }
   }
-  // tslint:disable-next-line: max-file-line-count
+
+  private sanitizeIQuery(query: IQuery): boolean {
+    if (!query || typeof query !== 'object' || Array.isArray(query)) {
+      return false;
+    }
+    if (!query || !Array.isArray(query.$or)) {
+      return false;
+    }
+    for (const item of query.$or) {
+      if (
+        typeof item._content_type_uid !== 'string' ||
+        typeof item.uid !== 'string' ||
+        (item._version && typeof item._version.$exists !== 'boolean') ||
```

Comment:
@harshithad0703 , what if $exists doesn't exists?

Reply:
will check other way for this

Reply:
```
interface IQuery {
  $or: Array < {
    _content_type_uid: string,
    _version?: {
      $exists: boolean,
    },
    uid: string,
    locale?: string,
  } >
}
```

this the type definition for IQuery